index.js

```diff
@@ -78,7 +77,6 @@ module.exports = {
             'beforeColon': false,
             'afterColon': true
         }],
-        'max-nested-callbacks': [2, 3],
```

Comment:
We see no real value to this. What's the reasoning?
Especially in tests this is annoying, but we realize this is prob the most "controversial" of our changes